packages/origin-ui/src/components/Account/OrganizationRegister.tsx

```diff
@@ -150,7 +153,14 @@ export function OrganizationRegister() {
         formikActions.setSubmitting(true);
         dispatch(setLoading(true));
 
-        showNotification('Organization registered.', NotificationType.Success);
+        try {
+            await axios.post(`${environment.BACKEND_URL}/api/Organization`, values);
```

Comment:
I think we should replace this with a implementation on `origin-backend-client` instead. This is hard to test among other problems.